src/cli.js

```diff
@@ -46,8 +46,10 @@ const runCli = (options) => {
         if (duplicatedPropertyKeys != null) {
           if (duplicatedPropertyKeys.length === 0) {
             console.log(chalk.green(`No duplicated property keys found in ${options.src}.`))
+            return 0;

```

Comment:
Please remove the semicolon, as these are not desired according to https://standardjs.com/ code conventions, which I am following here.